runbotics/runbotics-orchestrator-ui/src/components/ResizableDrawer/ResizableDrawer.tsx

```diff
@@ -1,24 +1,23 @@
-import React, { useState } from 'react';
+import React, { FC, useState } from 'react';
 import { Drawer } from '@mui/material';
 import { DrawerProps } from '@mui/material/Drawer/Drawer';
 import useWrappedState from 'src/hooks/useWrappedState';
-import { Dragger, EventCatcher } from './ResizableDrawer.styled';
 import { createPortal } from 'react-dom';
+import { Dragger, EventCatcher } from './ResizableDrawer.styled';
+import { DRAWER_WIDTH, MAX_DRAWER_WIDTH, MIN_DRAWER_WIDTH } from '../InfoPanel';
 
-const INITIAL_WIDTH = 300;
-const MIN_WIDTH = 250;
-const MAX_WIDTH = 600;
-
-const ResizableDrawer = ({ children, open, ...other }: DrawerProps) => {
-    const [width, setWidth] = useWrappedState(INITIAL_WIDTH, { min: MIN_WIDTH, max: MAX_WIDTH });
+const ResizableDrawer: FC<DrawerProps> = ({ children, open, ...other }) => {
+    const [width, setWidth] = useWrappedState(DRAWER_WIDTH, { min: MIN_DRAWER_WIDTH, max: MAX_DRAWER_WIDTH });
     const [dragmode, setDragmode] = useState(false);
 
     const handleDragStart = () => {
         setDragmode(true);
     };
 
     const handleMouseMove = (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
-        setWidth(window.innerWidth - event.clientX);
+        setTimeout(() => {
+            setWidth(window.innerWidth - event.clientX);
+        }, 100);
```

Comment:
why do we need timeout here?

Reply:
resize dragger was lagging on my local machine then I found it helpful to reduce the problem but tbh it is not a problem on dev or prod so I will undo